src/config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine MIXXX_INSTALL_DOCDIR_RELATIVE_TO_DATADIR "@MIXXX_INSTALL_DOCDIR_RELATIVE_TO_DATADIR@"
```

Comment:
interesting, I wasn't aware that we can use cmake vars in the actual project code.
somehow I considered those two entirely separate 'things'

Reply:
We can use them via configure_file or target_compile_definitions. The `#cmakedefine` is like a regular `#define` except that it maps to `undef` if its not set.